UPGRADING.md

```diff
@@ -0,0 +1,10 @@
+- [Upgrading Opensearch GO Client](#upgrading-opensearch-go-client)
+    - [Upgraading to >= 3.0.0](#upgrading-to->=-3.0.0)
+        - [opensearchapi](#opensearchapi-snapshot-delete)
+
+# Upgrading Opensearch GO Client
+
+## Upgraading to >= 3.0.0
+
+### opensearchapi snapshot delete
+SnapshotDeleteRequest and SnapshotDelete changed the argument `Snapshot` type from `string` to `[]string`.
```

Comment:
Maybe give a before/after example. I know it sounds like overkill but it's best to help users as much as we can.

Reply:
Updated the file.